app/models/current.rb

```diff
@@ -0,0 +1,18 @@
+class Current < ActiveSupport::CurrentAttributes
+  attribute :user
+
+  def account
+    return nil unless account_id.present?
```

Comment:
It might be better to raise from here if the `account_id` isn't present. It means you're looking for an account when not signed in. Alternatively, return a null account object with the same interface as real accounts so there's no error. We should lean hard one way or the other.